Model/User.php

```diff
@@ -88,7 +85,7 @@
     protected $passwordRequestedAt;
 
     /**
-     * @var Collection
+     * @var Array
```

Comment:
this cannot be an array. Doctrine will put a collection in it when loading the object.

btw, ``array`` should be lowercased.